group.py

```diff
@@ -418,8 +418,19 @@ def api_group_create(ctx, group_name, category, subcategory, description, data_c
     :param subcategory:         Subcategory of the group to create
     :param description:         Description of the group to create
     :param data_classification: Data classification of the group to create
+
+    :returns: Dict with API status result
     """
-    ctx.uuGroupAdd(group_name, category, subcategory, description, data_classification, '', '')
+    try:
+        response = ctx.uuGroupAdd(group_name, category, subcategory, description, data_classification, '', '')['arguments']
+        status = response[5]
+        message = response[6]
+        if status == '0':
+            return api.Result.ok()
+        else:
+            return api.Error('-1', message)
+    except Exception:
+        return api.Error('-1', 'Something went wrong creating group "{}". Please contact a system administrator'.format(group_name))
```

Comment:
Instead of -1 we can give the error a name to influence the http status returned by the portal (https://github.com/UtrechtUniversity/yoda-portal/blob/26b233cc613ba0227de482f4ac16c23ab25312a3/api.py#L30).

When an exception occurs, we can use the name 'error_internal'. This will result in http status 500.